### PR TITLE
PR: Add pyls-mypy recipe

### PR DESCRIPTION
**Pull request by steff456**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by andfoy** [comment]
@conda-forge/staged-recipes, this one is ready for review! I also accept being maintainer of this package.

cc @ccordoba12

File: recipes/pyls-mypy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
```
Comment (scopatz) [inline recipes/pyls-mypy/meta.yaml:5]: Please remove comments

**Comment by ccordoba12** [comment]
Me too.

File: recipes/pyls-mypy/LICENSE [hunk 2]
```diff
@@ -0,0 +1,22 @@
+
```
Comment (isuruf) [inline recipes/pyls-mypy/LICENSE:1]: I assume the License file is not packaged in the sdist. Is this fixed upstream?

Comment (steff456) [reply recipes/pyls-mypy/LICENSE:1]: The tar.gz doesn't contain the License file, but they have it in the repository https://github.com/tomv564/pyls-mypy/blob/master/LICENSE, should I remove that file from this PR?

Comment (chrisburr) [reply recipes/pyls-mypy/LICENSE:1]: The filename should be included in the `MANIFEST.in` file upstream so it's included in future versions. Without it the tarball violates this line of the license:

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

**Pull request merged**